### PR TITLE
added check for valid ayah

### PR DESCRIPTION
**Pull request by naveed-ahmad**
/1/11 => /error/invalid-ayah

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32808

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32809

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32810

File: src/containers/Error/index.js [hunk 1]
```diff
@@ -1,21 +1,25 @@
 import React, { PropTypes } from 'react';
 import IndexHeader from 'components/IndexHeader';
 import Helmet from 'react-helmet';
+import { FormattedHTMLMessage } from 'react-intl';
 
 const error = {
-  'invalid-surah': 'Surah is out of range',
-  'invalid-ayah-range': 'Ayah(s) selected are out of range'
+  'invalid-surah': "Surah is out of range. Please go to <a href='/'> home page</a> and select a Surah",
```
Comment (sabeurthabti) [inline src/containers/Error/index.js]: can we use `<Link to="/">`, we don't want to do a refresh.

Comment (mmahalwy) [inline src/containers/Error/index.js]: Not sure if that will work since it's text and not JSX. Need to change this to be react code for that to work

Comment (naveed-ahmad) [inline src/containers/Error/index.js]: @sabeurthabti @mmahalwy 
tried to convert this into JSX, got this.
Surah is out of range. Please go to [object Object] and select a Surah
```javascript
const homeLink = <Link to='/'> home page </Link>;
const error = {
  'invalid-surah': `Surah is out of range. Please go to ${homeLink} and select a Surah`,
  'invalid-ayah': `Ayah is out of range. Please go to ${homeLink} and select a Surah/Ayah`
};
```

**Pull request merged**